x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/inference/action/GetInferenceModelAction.java

```diff
@@ -105,10 +86,6 @@ public void writeTo(StreamOutput out) throws IOException {
             if (out.getTransportVersion().onOrAfter(TransportVersions.V_8_16_0)) {
                 out.writeBoolean(this.persistDefaultConfig);
             }
-
-            if (out.getTransportVersion().onOrAfter(TransportVersions.INFERENCE_MODEL_REGISTRY_METADATA)) {
-                out.writeBoolean(returnMinimalConfig);
-            }
```

Comment:
This is a leftover from #121106. It's ok to remove since the transport serialisation is not used on `HandledTransportAction` that executes directly on the receiving node.